### DISCUSSION TITLE
Is there  any possible scene that kvrocks cluster may cause lost data ?  how to fix?

### DISCUSSION THREAD

**Discussion by jcy1001** [apache/kvrocks#1670] [General]
Is there  any possible scene that kvrocks cluster may cause lost data ?  how to fix?
1: when one master down ,the slave promoted ,may lost 2s data ?

**Comment by git-hulk** [comment]
@jcy1001 Kvrocks replication is the eventual consensus, so I don't think it's possible to mitigate this situation for now.

**Reply by bbarani** [reply]
Is there a plan to introduce semi-synchronous / consensus replication model for KV rocks?
